src/style.css

```
@@ -0,0 +1,90 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir, Helvetica, sans-serif;
+}
+
+body {
+  margin-top: 0;
+  --boder-color: #e1e7ed;
+}
+
+.bar-form {
+  width: 100%;
+}
+
+.field {
+  position: relative;
+  height: 70px;
+  width: 100%;
+  margin-top: 20px;
+}
+
+.small-field {
+  height: 42px;
+  width: 100%;
+}
+
+.image {
+  position: absolute;
+  left: 26px;
+  top: 25px;
+  display: block;
+  height: 19px;
+  width: 19px;
+}
+
+.image-small {
+  left: 13px;
+  top: 15px;
+  height: 11px;
+  width: 11px;
+}
+
+.search-bar {
+  position: absolute;
+  box-sizing: border-box;
+  width: 100%;
+  height: 100%;
+  padding: 0 0 0 62px;
+  font-size: 16px;
+  color: #3d4e61;
+  border-radius: 4px;
+  background: #fff;
+  border: 1px solid var(--boder-color);
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  outline: none;
+}
+
+.small-search-bar {
+  width: 100%;
+  height: 100%;
+  padding: 0 0 0 33px;
+  font-size: 14px;
+  padding-bottom: 2px;
+}
+
+.field:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-bar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid var(--boder-color);
+  border-radius: 3px;
+}
+
+.search-bar::placeholder {
+  font-family: Avenir, Helvetica, sans-serif;
```

```suggestion
  font-family: inherit;
```

It should work as well